libs/features/personalization/personalization.js

```diff
@@ -561,13 +561,25 @@ export const deleteMarkedEls = (rootEl = document) => {
     .forEach((el) => el.remove());
 };
 
+export function addSectionAnchors(rootEl = document) {
+  rootEl.querySelectorAll('.section-metadata').forEach((block) => {
+    [...block.children].forEach((row) => {
+      const col1 = row.children[0]?.textContent.toLowerCase().trim();
+      const col2 = row.children[1]?.textContent.toLowerCase().trim().replaceAll(' ', '-');
```

Comment:
```suggestion
      const col2 = row.children[1]?.textContent.toLowerCase().trim().split(/\s+/).join('-');
```
couldn't get replaceAll to work with a regex, but maybe I'm just doing it wrong lol

Reply:
Should work
```
'abc def  ghi   jkl    mno     p'.replaceAll(/\s+/g, '-');
```

Reply:
Updated just in case, but it looks like EDS transforms multiple whitespaces to just one 
![Screenshot 2025-03-07 at 11 30 02](https://github.com/user-attachments/assets/9ea76ca5-2df7-4a9a-aef8-fcba22dacb67)
![Screenshot 2025-03-07 at 11 30 21](https://github.com/user-attachments/assets/ff6e3b0c-2bc3-48cc-b413-cdc1f585f02b)
@chrischrischris